libraries/core_libs/network/src/tarcap/packets_handlers/latest/get_pbft_sync_packet_handler.cpp

```diff
@@ -81,6 +85,21 @@ void GetPbftSyncPacketHandler::sendPbftBlocks(const std::shared_ptr<TaraxaPeer>
       return;
     }
 
+    // Add pillar votes to period data
+    const auto &ficus_hf_conf = kConf.genesis.state.hardforks.ficus_hf;
+    if (ficus_hf_conf.isPillarBlockPeriod(block_period, 2)) {
```

Comment:
What is this 2? Looks like it should be some constant somewhere

Reply:
fixed